SmartSport guide completed

Atlantido opened the pull request.

Atlantido committed:
Update smartspot.md without images

Atlantido committed:
created images folder

Atlantido committed:
uploaded images

Atlantido committed:
deleted prueba.md

Atlantido committed:
deleted ESP32 DevKitC

Atlantido committed:
uploaded ESP32 DevKitC

Atlantido committed:
uploaded sensors

Atlantido committed:
uploaded connection

Atlantido committed:
Update smartsport.md with images

Atlantido committed:
Actualizar fichero

Atlantido committed:
Update smartsport.md

Atlantido committed:
Update smartsport

All links is working

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+Optionally, this system provides information about crowds in specific areas, detecting people with smartphone WiFi switched-on.
+
+
+## **What do you need?**
```
please use canonical markdown

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+# User&#39;s guide for Smart Spot Starter Kit
 
-## What you need to order
+## **Introduction**
```
please use canonical markdown

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+## **Introduction**
 
-## How to install and configure it
+In order to offer a simple and intuitive way to get started with our Smart Spot we have developed an Open Source solution called Smart Spot Starter Kit, which has an obvious educational purpose.  Just by simply obtaining this kit and follow this guide you will be able to manage the device and to acquire data from its board sensors, apart from the ones you connect, with your smartphone.
```
apply linter, lines should me max 80 chars, see for applied rules:
https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md

and here for the linter detected issues: https://travis-ci.org/smartsdk/guided-tour/jobs/346396767

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+
+## **What do you need?**
+
```
where can it be ordered?

imagine people really need to build the system, how that will be possible?

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+## **What do you need?**
+
+### What is included with The Smart Spot Starter Kit:
```
this level three is not needed given that there are no other level three in same section

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+![ESP32 DevKitc](./images/ESP32-DevKitC.png)
+
+2. Starter Kit firmware
+
```
use canonical md, start always with 1. and use proper indentation to ensure numbering is accounted correctly

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+1. ESP32 DevKitC.
+
+![ESP32 DevKitc](./images/ESP32-DevKitC.png)
+
```
picture should be intended

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+6. I2C Cable
+ 
+![I2C Cable](./images/I2C.png)
+
```
picture should be indented

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+ 	* Nitrogen Dioxide (Orange).
+
+![Gas Sensor](./images/sensor.png)
+
```
picture should be indented

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+[http://esp-idf.readthedocs.io/en/latest/get-started/index.html](http://esp-idf.readthedocs.io/en/latest/get-started/index.html)
+
+
+1) First, you need to download the ESP32 toolchain by clicking in this link: [https://dl.espressif.com/dl/esp32\_win32\_msys2\_environment\_and\_toolchain-20180110.zip](https://dl.espressif.com/dl/esp32_win32_msys2_environment_and_toolchain-20180110.zip). Unzip it and place somewhere safe. This guide assumes the environment is placed in _C:\ _ .
```
use canonical md numbered list.

```
1. xxx
```

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+2) You will also need the ESP32 API. In order to download it, open a terminal, navigate to a directory where you want to place the IDF and use these commands:
+
+~~~
```
use canonical md code and indent it for proper alignment below the list

1. xxxx

    ```bash
    $ git clone --recursive https://github.com/espressif/esp-idf.git
 
    $ git checkout tags/v3.0-rc1
 
    $ git submodule update –init
    ```

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+$ git submodule update –init
+
+~~~
```
see above

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+~~~
+
+$ export IDF\_PATH=&quot;C:/msys32/home/user-name/esp/esp-idf&quot;
+
```
see above

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+~~~
+
+$ git clone –-https://github.com/HOP-Ubiquitous/SmartSpot\_SmartSDK\_Firmware.git
+
```
see above

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+~~~
+
+$ python /c/Users/HOPU/GitHub/espidf/components/esptool\_py/esptool/esptool.py --chip esp32 --port **COMX** --baud 115200 --before default\_reset --after hard\_reset write\_flash -z --flash\_mode dio --flash\_freq 40m --flash\_size detect 0x1000 **C:/Users/HOPU/GitHub/SmartSpot\_SmartSDK\_Firmware/bootloader.bin** 0x10000
+
```
see above

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+A message like the one above should appear if you flashed the firmware successfully.
+
+## **Expansion board integration**
+
```
use canonical md

chicco785 commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+![Connection](./images/connection.png)
+
+## **Fiware integrations**
+FIWARE ([www.fiware.org](http://www.fiware.org)) is an open platform promoted by the European Commision and maintained by the FIWARE Foundation, where HOP Ubiquitous is Gold Member.
```
this is a tutorial, so this text is not appropriate,
just shortly describe how to connect the sensor to the proper IoT Agent.

you should also describe how to register a url to be use for physical web interaction

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+You will need a PC loaded with Windows, Linux or Mac OS with internet connection in order to download the Starter Kit firmware and the ESP32 toolchain and ESP-IDF.
+
+## **Getting Started**
```
I'd suggest adding (windows only) here

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,199 @@
+
+## **Register your Smart Spot in the IoT Agent**
+
+First, you need to create a WiFi AP with your smartphone for the first connection of the SmartSpot. Once you register in the IoT Agent, you can change the WiFi parameters. This are the default SSID and password:
```
There are missing (assumed) steps here, please either add info or links to where that is documented (create ap, register iota, etc)

dancn commented:
Hello,
small suggestion: for the MarkDown compliance checking you can use this one-liner:
https://github.com/smartsdk/guided-tour/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R29

Atlantido committed:
Update smartspot.md

fixed part of the document

Atlantido committed:
update smartspot.md

Fixed part of the document Version 2

Atlantido committed:
Update smartspot.md

Fixed part of the document

Atlantido committed:
update smartspot.md

Fixed part of the document

Atlantido committed:
Update smartSpot.md

Fixed fiware integrations

Atlantido committed:
Update smartsport.md

Expansion board integration, Fiware integrations, Register your Smart Spot in the IoT Agent and Gas Sensor Integration fixed

Atlantido committed:
Update smartspot.md

All images has centered alignment and all errors are fixed

Atlantido committed:
Update smartdown.md

Atlantido committed:
Update smartspot.md

Atlantido committed:
Update smartspot.md

Atlantido committed:
Update smartspot.md

Atlantido committed:
Update smartspot.md

Atlantido committed:
Update smartspot.md

taliaga commented:
Hi @Atlantido, content-wise, the changes LGTM.
Last thing I'd suggest is having a look a the CI issues (https://travis-ci.org/smartsdk/guided-tour/jobs/364016350).
Thanks!

chicco785 commented:
@Atlantido please lint the markdown as by CI report https://travis-ci.org/smartsdk/guided-tour/jobs/364016350

thx

Atlantido commented:
@chicco785 we are adding new content to guide. When we finish adding the new content then we will do pull request

chicco785 commented:
hi, by when? we want this to be online before the summit next week :)

thx

Atlantido commented:
Tomorrow is a holiday in Spain and this Wednesday in the afternoon or first thing on Monday we will do the pull request

Atlantido committed:
Update smartdown.md

Atlantido committed:
Minor fixes

Atlantido committed:
Update smartspot.md

Atlantido committed:
minor fixes

Atlantido committed:
upload images

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,982 @@
+readme file of that repository, a complete guide to deploy the docker
+infrastructure can be found.
+
+~~~
```
As promised, I'm checking the latest changes. Some comments here:
- I'd put this huge config in a file and reference from it. It bloats the document unnecessarily. If the config already lives somewhere else, just reference it.
- Use single backticks to wrap special words such as environment variable names or filenames: `config.js`
- Minor, but I've never seen those ~~~ to indicate code blocks, I knew of ``` . Does that render fine in the generated docs?. (see a pointer in the README at the root of this project for the doc builder tool)

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,982 @@
+      available. The specified port in the server section, needs to be the
+      same here.
+
+In the following table you can find a config.js file as example for the FIWARE
```
table?

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,982 @@
+
+### Build architecture through docker-compose
+
+Please, take a look to the docker-compose.yml file, in this one you can find the
```
Readers will not know where that file is, will they?

taliaga commented on docs/sensors/smartspot.md:
```diff
@@ -1,11 +1,982 @@
+
+## To Be Done
+
+Add FIWARE security stack Create different docker-composes adapting the
```
may read better as a list (or bracketed list) as below:
- AA
- BB
- [ ] A
- [ ] B

Atlantido committed:
minor fixes

Atlantido committed:
minor fixes

Atlantido committed:
update smartspot.md

Atlantido committed:
update smartspot.md

Atlantido committed:
update smartspot.md

Atlantido committed:
Update smartspot.md

Atlantido committed:
delete sensors.png

Atlantido committed:
add images

Atlantido committed:
delete sensors,png

Atlantido committed:
add image

Atlantido committed:
config.js added

Atlantido committed:
minor fixes

Atlantido commented:
@taliaga In your requested changes you said "may read better as a list (or bracketed list)". I don´t understand what you mean with it can be read better as a list.

Atlantido committed:
delete board.png

Atlantido committed:
Delete sensor.png

Atlantido committed:
delete connection.png

Atlantido committed:
Images added

Atlantido committed:
delete board.png

Atlantido committed:
delete connection.png

Atlantido committed:
delete sensor.png

Atlantido committed:
add files

Atlantido committed:
minor fixes
